src/pages/deriv-go/_banner.tsx

```diff
@@ -150,25 +150,25 @@ const HeroContent = styled(Flex)`
         }
     }
 `
+
 const Banner = () => {
     const data = useStaticQuery(query)
     const [is_mobile, setMobile] = useState(false)
-    const handleResizeWindow = () =>
-        setMobile(isBrowser() ? window.screen.width <= size.tablet : false)
 
     useEffect(() => {
+        const handleResizeWindow = () =>
+            setMobile(isBrowser() ? window.screen.width <= size.tablet : false)
         handleResizeWindow()
         window.addEventListener('resize', handleResizeWindow)
 
         return () => {
             window.removeEventListener('resize', handleResizeWindow)
         }
-    }, [handleResizeWindow])
+    }, [is_mobile])
 
     const handleExternalLink = () => {
         let link = deriv_go_playstore_url
         if (is_mobile) {
-            // TODO handle IOS case once the app is ready
```

Comment:
What is the reason behind removing this TODO ?

Reply:
Action already declared. 2 weeks ago. Developer just forgot to remove that
<img width="399" alt="Screenshot 2022-07-25 at 16 55 39" src="https://user-images.githubusercontent.com/103182473/180796204-b548aaa5-a4ef-472f-8300-f33a21679a6a.png">